test/alternator/suite.yaml

```diff
@@ -1 +1,23 @@
-type: Run
+type: Python
+pool_size: 4
+extra_scylla_cmdline_options:
+#   - '--alternator-address=8080'
+    - '--alternator-enforce-authorization=1'
+    - '--alternator-write-isolation=always_use_lwt'
+    - '--alternator-streams-time-window-s=0'
+    - '--alternator-timeout-in-ms=30000'
+    - '--alternator-ttl-period-in-seconds=0.5'
+#    - '--alternator_port=8000'
+  # Allow testing experimental features. Following issue #9467, we need
+  # to add here specific experimental features as they are introduced.
+  # We only list here Alternator-specific experimental features - CQL
+  # ones are listed in test/cql-pytest/run.py.
+#    - '--experimental-fa   eatures=alternator-streams'
+run_first:
+    - test_streams
+    - test_scan
+    - test_tracing
+    - test_ttl
+extra_scylla_config_options:
+  { alternator_port: 8000,
+    alternator_write_isolation: only_rmw_uses_lwt}
```

Comment:
From all the unnecessary conftest.py changes I saw above, I missed this, which is the important part of this patch :-) This is what I thought all along we need to do, and only this. 
@temichus is right that for the authentication tests, we also need that extra authetication-writing CQL statement, and instead of hard-coding it in test.py (or conftest.py) maybe we can add a new variable in suite.yaml which invokes some additional setup function on the cluster or something.